webpack.config.js

```diff
@@ -3,13 +3,9 @@ const HtmlWebpackPlugin = require("html-webpack-plugin");
 const MiniCssExtractPlugin = require("mini-css-extract-plugin");
 
 const DEFAULT_TRAINING_MATERIAL_FOLDER = "training-material";
+const DEFAULT_LANG = "en";
 
 module.exports = (env = {}, argv) => {
-  if (!env.material) {
-    console.warn(
-      `WARNING: '--env.material' option not set. Falling back on the default: '${DEFAULT_TRAINING_MATERIAL_FOLDER}'`
-    );
-  }
```

Comment:
Why is this removed?

Reply:
I think the default behavior is fine and warning is not necessary. There is also already a log that inform the user which material sensei is serving and how to override it

Reply:
OK, I did not realize this.